src/components/cards/index.css

```diff
@@ -0,0 +1,82 @@
+
+/* .overlay-card{
```

Comment:
remove the commented code , if that is there just for your reference for right now then it is good.